### PR TITLE
October npm update

### PR DESCRIPTION
**Pull request by illepic**

### CODE REVIEW

File: source/default/_patterns/02-molecules/carousel/_carousel.twig [hunk 1]
```diff
@@ -51,3 +50,5 @@
     {% endblock controls %}
   </div>
 </div>
+
+
```
Comment (Jwaxo) [inline source/default/_patterns/02-molecules/carousel/_carousel.twig]: Extra newline if you feel like removing.

File: source/default/_patterns/02-molecules/carousel/_carousel.twig [hunk 2]
```diff
@@ -26,8 +26,7 @@
     {% for image in carousel_images %}
       <div class="carousel-item {{ (image.active) ? 'active' }}">
         {% include '@atoms/image/_image.twig' with {
-          imageStyleName: 'carousel',
-          class: 'img img-fluid',
+          image: image,
```
Comment (Jwaxo) [inline source/default/_patterns/02-molecules/carousel/_carousel.twig]: Yesssssss.

File: source/default/_patterns/02-molecules/carousel/_carousel.twig (hunk 1 above)
Comment (illepic) [reply source/default/_patterns/02-molecules/carousel/_carousel.twig]: Good catch!

File: apps/node-pl/pl-app.js [hunk 3]
```diff
@@ -27,10 +27,6 @@ pl.events.on('patternlab-build-start', () => console.log(`${message} START!`));
...
 /**
- * patternsOnly in dev, full build in prod
+ * Full build (including ui)
  */
-if (NODE_ENV === 'development') {
-  pl.patternsonly(options);
-} else {
-  pl.build(options);
-}
+pl.build(options);
```
Comment (illepic) [inline apps/node-pl/pl-app.js]: We can revisit this. I fell back to just a full rebuild on change to ensure new components always show.

**Pull request merged**